Description: Upstream fixed a bug where adding multiple files by
 drag and drop only added the first file.
Origin: upstream, http://code.google.com/p/winff/source/detail?r=592
Author: Paul Gevers <paul@climbing.nl>

Index: winff-git/unit1.pas
===================================================================
--- winff-git.orig/unit1.pas	2011-12-17 14:04:48.364445081 +0100
+++ winff-git/unit1.pas	2011-12-17 14:05:07.748541229 +0100
@@ -1304,6 +1304,7 @@
 begin
 numfiles := high(Filenames);
 for i:= 0 to numfiles do
+begin
    s :=FileNames[i]; // fix for 1.4 (was using filenames from filelistbox)
    u := s;
    //t := GetFileInfo(u); // 1.4 not needed now
@@ -1314,6 +1315,7 @@
    JobList.add(t);
    FileInfoList.add(u);
 end;
+end;
 
 // add files to the list
 procedure TfrmMain.btnAddClick(Sender: TObject);
